src/App.js

```diff
@@ -30,36 +32,130 @@ const App = () => {
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
 
-  // Wave 2
-  // You will need to create a method to change the square
-  //   When it is clicked on.
-  //   Then pass it into the squares as a callback
-
-  const checkForWinner = () => {
-    // Complete in Wave 3
-    // You will need to:
-    // 1. Go accross each row to see if
-    //    3 squares in the same row match
-    //    i.e. same value
-    // 2. Go down each column to see if
-    //    3 squares in each column match
-    // 3. Go across each diagonal to see if
-    //    all three squares have the same value.
+  const [playerOnesTurn, setTurn] = useState(false);
+
+  // togglesSquares
+  const onClickCallback = (id) => {
+    let player;
```

Comment:
There is a small logical bug here.  There is no check here to see if there is a winner, so it is possible to continue to play.  This can result in unexpected behavior.  For example, if the board looks like this:
```
| | | |
| |o|o|
|x|x|x|
```
the game will display 'Winner is x'.  However, 'o' can still move, so if 'o' clicks in the middle left square:
```
| | | |
|o|o|o|
|x|x|x|
```
the game will display "Winner is o".